amethyst_rendy/src/debug_drawing.rs

```diff
@@ -96,6 +96,207 @@ impl DebugLinesComponent {
         self.lines.push(vertex);
     }
 
+    /// Adds multiple lines that form a square to be rendered by giving a z coordinate, a min and a max position.
+    pub fn add_square(&mut self, min: Point2<f32>, max: Point2<f32>, z: f32, color: Srgba) {
```

Comment:
i would name that `add_square_2d` or at least add something to the docs to make it clear that it behaves "like sprites", i.e. it's aligned to XY plane.